root/components/FieldErrors.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+import subfieldErrors from '../utility/subfieldErrors';
+
+const buildErrorListItem = (error, index) => (
+  <li key={index}>{error}</li>
+);
+
+type Props = {|
+  +field: AnyFieldT,
+|};
+
+const FieldErrors = ({field}: Props) => {
+  if (!field) {
+    return null;
+  }
+  const errors = subfieldErrors(field);
+  if (errors.length) {
+    return (
+      <ul className="errors">
+        {errors.map(buildErrorListItem)}
+      </ul>
+    );
+  }
+  return null;
+};
+
+export default FieldErrors;
```

Comment:
Why not export it with the definition?

Reply:
`export default {var,let,const}` isn't valid syntax, though I can change it a named function and do that.

Reply:
But not for components using an implicit return (can't do that with named functions). I think I'll keep them consistent.